swift/swift.py

```diff
@@ -33,93 +116,77 @@ class Swift(QObject):
         3. Create apps and show their frames.
     """
 
-    def __init__(self, setupEnv: dict, parent=None):
+    def __init__(
+        self,
+        appInfos: Mapping[str, AppInfo] | None = None,
+        busInfos: Mapping[str, BusInfo] | None = None,
+        parent: QObject | None = None):
         """
         Args:
-            setupEnv: A dictionary containing set-up environment about app and bus.
-              It has two keys; "app" and "bus".
-              In "app", there are apps containing keys as below; 
-                "path" (optional), "module", "class", "show", "pos", "bus", and "args" (optional).
-              In "bus", there are buses containing keys as below;
-                "timeout" (optional).
-              For details, see setup.json.
+            appInfos: See Swift.load(). None or an empty dictionary for loading no apps.
+            busInfos: See Swift.load(). None or an empty dictionary for loading no buses.
             parent: A parent object.
         """
         super().__init__(parent=parent)
         self.mainWindow = QMainWindow()
         self._buses = {}
         self._apps = {}
         self._subscribers = {}
-        self.load(setupEnv["app"], setupEnv["bus"])
+        appInfos = appInfos if appInfos else {}
+        busInfos = busInfos if busInfos else {}
+        self.load(appInfos, busInfos)
         self.mainWindow.show()
 
-    def load(self, appInfos: dict, busInfos: dict):
+    def load(self, appInfos: Mapping[str, AppInfo], busInfos: Mapping[str, BusInfo]):
         """Initializes swift system and loads the apps and buses.
         
         Args:
-            appInfos: See "app" part of setupEnv in self.__init__().
-            busInfos: See "bus" part of setupEnv in self.__init__().
+            appInfos: A dictionary whose keys are app names and the values are
+              corresponding AppInfo objects. All the apps in the dictionary
+              will be created and shown if the show field is True.

```

Comment:
It looks a little ambiguous since it may be interpretted that 'will be created if the show field is True'.

Is there any better explanation..?